cdbot/cogs/admin.py

```diff
@@ -89,6 +92,32 @@ async def on_member_join(self, member: Member):
             # assign placeholder nickname
             await member.edit(nick=PLACEHOLDER_NICKNAME)
 
+    @Cog.listener()
+    async def on_message(self, message):
+        def check(m):
```

Comment:
Check? Check for what? Give this function a name that tells me what it actually is; it's performing a very specific task, so name it like it is.

Reply:
Docstrings, typehints, etc.